yt_dlp/extractor/manoto.py

```diff
@@ -11,25 +11,50 @@ class ManotoTVIE(InfoExtractor):
         'url': 'https://www.manototv.com/episode/12576',
         'info_dict': {
             'id': '12576',
-            'title': 'Seh Mah Taatili',
-            'description': 'مجموعه ای از فیلم های سینمای کلاسیک ایران',
+            'series': 'فیلم های ایرانی',
+            'season_number': 0,
+            'episode_number': 0,
+            'episode_id': 'Seh Mah Taatili',
+            'duration': 5400,
+            'view_count': 10550,
+            'categories': ['سرگرمی'],
+            'title': 'سه ماه تعطیلی',
+            'description': 'سه ماه تعطیلی فیلمی به کارگردانی و نویسندگی شاپور قریب ساختهٔ سال ۱۳۵۶ است.<br/><br/>',
```

Comment:
the html tags should be removed by passing through cleanhtml

Reply:
Done.